www/PoyntPlugin.js

```diff
@@ -12,6 +12,50 @@ var Poynt = {
 				"amount": amount, 
 				"referenceId": referenceId
 			}]);
+	},
+	launchBilling: function (paymentid, successCallback, errorCallback){
+		cordova.exec(successCallback, 
+			errorCallback, 
+			'Poynt', 
+			'launchBilling', 
+			[{
+				"paymentid": paymentid
+			}]);
+	},
+	launchAskConf: function (msg,successCallback, errorCallback){
+		cordova.exec(successCallback, 
+			errorCallback, 
+			'Poynt', 
+			'launchAskConf', 
+			[{"msg": msg}]);
+	},
+	launchSign: function (title,butt,msg,successCallback, errorCallback){
```

Comment:
similarly, put this inside `secondScreen` object and rename the function to `collectSignature` to be consistent with http://bit.ly/2M0QdWQ. `title`, `rightButtonTitle`, `textUnderLine`, `amount`, `currency`, `tipAmount` should go into `options` object.